modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -1769,7 +1770,95 @@ def sort_by(
         PandasDataframe
             A new PandasDataframe sorted into lexicographical order by the specified column(s).
         """
-        pass
+        columns = columns if isinstance(columns, list) else [columns]
+        axis = Axis(axis)
+        if axis == Axis.ROW_WISE and not (
+            len(columns) == 1 and columns[0] == self.index.names[0]
+        ):
+
+            def sample_func(df):
```

Comment:
could you add a short docstring to these funcs, just to briefly explain what they do?